### PR TITLE
Feature form validation

### PR DESCRIPTION
**Pull request by lincoln1883**
- Add javascript for validating email.
- Add Textarea attributes and remove the button hover style.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -231,3 +231,29 @@ for (let i = 0; i < projectButtons.length; i += 1) {
...
+
+form.appendChild(error);
+const button = form.querySelector('button');
+form.insertBefore(error, button);
+
+error.classList.add('error');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const isValid = !regex.test(email.value);
```
Comment (LionRouge1) [inline script.js:249]: - [ ] Good job implementing the email validation :clap: However the form is not submitting when the email field contain only lowercase. Kindly check it.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @lincoln1883, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR :heavy_check_mark: 
- Good commit :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lincoln1883** [comment]
- Applied changes to accomplish functionality.

**Review by LionRouge1 — APPROVED** [review]
Hi @lincoln1883, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**